packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -62,9 +64,19 @@ export class ChainService implements ChainServiceInterface {
   private channelToSubscription: Map<Bytes32, Subscription[]> = new Map();
   private nitroAdjudicator: Contract;
 
+  private readonly blockConfirmations: number;
   private transactionQueue = new PQueue({concurrency: 1});
 
-  constructor({provider, pk, allowanceMode, pollingInterval}: ChainServiceArgs) {
+  constructor({
+    provider,
+    pk,
+    pollingInterval,
+    logger,
+    blockConfirmations,
+    allowanceMode,
+  }: ChainServiceArgs) {
+    this.blockConfirmations = blockConfirmations || 5;
```

Comment:
0 is a valid blockConfirmations value.
```suggestion
    this.blockConfirmations = blockConfirmations ?? 5;
```